components/telemetry-operator/README.md

```diff
@@ -68,59 +68,59 @@ make deploy-local IMG_NAME=<my container repo> TAG=latest
-
-```bash
-kyma deploy --component logging --value global.telemetry.enabled=true
-```
-
-The above command would additionally install a LogPipeline CR which would configure the telemetry-fluent-bit to push logs to the Loki. After disabling the fluent-bit from logging chart we need delete the unneeded resources
-
-```bash
-kubectl delete daemonset -n kyma-system logging-fluent-bit
-kubectl delete configmap -n kyma-system logging-fluent-bit
-kubectl delete servicemonitor -n kyma-system logging-fluent-bit
-```
-
-
-### Disabling Telemetry Operator integration
-```bash
-kyma deploy --component logging 
-```
-
-The above command would redeploy the fluent-bit from the logging chart. After installing the chart we can disable the LogPipeline CR so that it does not collect the logs anymore
-
-```bash
-kubectl delete validatingwebhookconfigurations validation.webhook.telemetry.kyma-project.io
-kubectl delete servicemonitor -n kyma-system telemetry-operator-metrics
-kubectl delete deployment -n kyma-system telemetry-operator
-kubectl delete daemonset -n kyma-system telemetry-fluent-bit
-kubectl delete service -n kyma-system telemetry-operator-webhook
-kubectl delete service -n kyma-system telemetry-operator-metrics
-kubectl delete service -n kyma-system telemetry-fluent-bit
-kubectl delete rolebinding -n kyma-system telemetry-operator-leader-election-rolebinding
-kubectl delete clusterrolebinding telemetry-operator-manager-rolebinding
-kubectl delete clusterrolebinding telemetry-fluent-bit
-kubectl delete clusterrole telemetry-operator-manager-role
-kubectl delete clusterrole logpipeline-viewer-role
-kubectl delete clusterrole logpipeline-editor-role
-kubectl delete clusterrole telemetry-fluent-bit
-kubectl delete configmap -n kyma-system telemetry-fluent-bit
-kubectl delete secret -n kyma-system telemetry-operator-webhook-cert
-kubectl delete serviceaccount -n kyma-system telemetry-operator
-kubectl delete serviceaccount -n kyma-system telemetry-fluent-bit
-```
\ No newline at end of file
+1. To install the Telemetry Operator together with the Telemetry Fluent Bit Daemon Set with null output config, run: 
+
+   ```bash
+   kyma deploy --component=telemetry 
+   ```
+
+2. Configure the Telemetry Fluent Bit component to push logs to the Loki backend. Loki comes with the Kyma out of the box .
+
+   ```bash
+   kyma deploy --component logging --value global.telemetry.enabled=true
+   ```
+
+   The previous command also installs a Log Pipeline CR, which configures the Telemetry Fluent Bit to push logs to the Loki backend.
+
+3. After disabling Fluent Bit from the logging chart, delete the unneeded resources. Run:
+
+   ```bash
+   kubectl delete daemonset -n kyma-system logging-fluent-bit
+   kubectl delete configmap -n kyma-system logging-fluent-bit
+   kubectl delete servicemonitor -n kyma-system logging-fluent-bit
+   ```
+
+### Disable Telemetry Operator integration
+
```

Comment:
Can we introduce this command with a sentence explaining its purpose? It installs the default logging component again without Telemetry Operator, right?

```suggestion
1. To disable the Telemetry Operator, simply deploy the default version of the logging component, which doesn't contain the Telemetry Operator:
```

Reply:
Applied with some modifications